network-api/networkapi/buyersguide/templates/product_page.html

```diff
@@ -47,88 +44,69 @@ <h2 class="h1-heading col-md-10">{{product.name}}</h2>
         </div>
 
-        <h3 class="h3-heading">What does it know about me</h3>
+        <h3 class="h3-heading">How it handles privacy</h3>
 
         <div class="mb-5">
-          {% include "fragments/product_criterion.html"  with  value=product.uses_encryption  bad=False  help=product.uses_encryption_helptext  label="Product uses encryption" criterion_name="uses-encryption" %}
-          {% include "fragments/product_policy.html"     with  product=product                           help=product.privacy_policy_helptext %}
-          {% include "fragments/product_criterion.html"  with  value=product.share_data       bad=True   help=product.share_data_helptext        label="Shares your information with 3rd parties for unexpected reasons" %}
+          {% include "fragments/product_criterion.html"  with  value=""                           help=product.how_does_it_share             label="How does it share data?"                %}
+          {% include "fragments/product_criterion.html"  with  value=product.delete_data          help=product.delete_data_helptext          label="Can you delete your data?"              %}
+          {% include "fragments/product_criterion.html"  with  value=product.parental_controls    help=product.child_rules_helptext          label="Parental controls?"                     %}
+          {% include "fragments/product_criterion.html"  with  value=product.collects_biometrics  help=product.collects_biometrics_helptext  label="Collects biometrics data?"              %}
+          {% include "fragments/product_criterion.html"  with  value=product.user_friendly_privacy_policy  help=""                           label="User friendly privacy info?"            %}
+
+          {% if product.privacy_policy_links|length > 0 %}
+          <ul class="policy-links">
+            {% for link in product.privacy_policy_links %}
+            <a class="body-large d-block" href="{{ link.url }}">{{ link.label }}</a>
+            {% endfor %}
+          </ul>
+          {% endif %}
         </div>
 
-        <h3 class="h3-heading">Can I control it</h3>
-
-        <div class="mb-5">
-          {% include "fragments/product_criterion.html"  with  value=product.strong_password    bad=False  help=product.strong_password           label="If a password is required, you must change the default password"  unknown_label="Not required" %}
-          {% include "fragments/product_criterion.html"  with  value=product.security_updates   bad=False  help=product.security_updates_helptext label="Automatic security updates" %}
-          {% include "fragments/product_criterion.html"  with  value=product.delete_data        bad=False  help=product.delete_data_helptext      label="Deletes data it stores on you" %}
-          {% include "fragments/product_criterion.html"  with  value=product.parental_controls  bad=False  help=product.child_rules_helptext      label="Has parental controls "%}
+        <div class="NEW_WORST_CASE_STYLING">
```

Comment:
@mavis this one will probably need HTML updating as well for the updated styling to work, so I just boxed it in with a div with a temporary class.